### DISCUSSION TITLE
Apple Silicon Compatibility Question

### DISCUSSION THREAD

**Discussion by AndrewFalkowski** [idaholab/moose#20811] [Q&A Getting Started] [answered]
I am in the process of spec'ing a mac for running moose simulations. I am wondering if there are any compatibility issues or caveats with apple silicon (M1) chips that I should understand or be aware of? My simulations are not particularly taxing - primarily 1D coupled diffusion processes. Thanks in advance!

**Comment by milljm** [comment, marked as answer]
I believe this PR may detail the caveats: https://github.com/idaholab/moose/pull/20349. For which the results of this PR can be found here: https://civet.inl.gov/job/1017127/

For the most part, Apple Silicon is working, for solving finite element problems using MOOSE. Some of the more 'pretty' features are not at the moment. Such as using the MOOSE GUI; Peacock. Peacock requires Qt. Qt at the moment is not officially supported on ARM (ref: https://www.qt.io/blog/qt-on-apple-silicon). And therefore not available in Conda-Forge.

**Reply by milljm** [reply]
If you do wish to use the packages currently available for ARM, they are available so long as you install one of the ARM variants of Miniforge/Mambaforge (we recommend [Mambaforge](https://github.com/conda-forge/miniforge#mambaforge)). Just swap out our current 'Getting Started' instructions with a link to an ARM Mambaforge, and that should be all you need to do.

**Reply by AndrewFalkowski** [reply]
Gotcha, any advice in terms of minimum RAM or processor requirements?

**Reply by GiudGiud** [reply]
For 1D simulations you should be good with the base amount of memory

**Reply by milljm** [reply]
As for processors... concerning the folks within my group; we always go with the most cores possibly available. Not necessarily speed, but core count. As for the Apple Silicon, because there is really only one or two options, this point is moot. M1, M1Pro, M1Max... But I think the Max is more a GPU core count upgrade than CPU one.

Anyone want to chime in? I could be wrong.

**Reply by GiudGiud** [reply]
Yeah no need to spend more on GPU so far. MOOSE GPU support is increasing but not there yet. 
Again for 1D simulations I think you dont need too much on CPU either, base option should do
